pom.xml

```diff
@@ -0,0 +1,440 @@
+            <id>nexus-releases</id>
+            <name>WSO2 Release Distribution Repository</name>
+            <url>http://maven.wso2.org/nexus/service/local/staging/deploy/maven2/</url>
+        </repository>
+        <snapshotRepository>
+            <id>wso2.snapshots</id>
+            <name>Apache Snapshot Repository</name>
+            <url>http://maven.wso2.org/nexus/content/repositories/snapshots/</url>
+        </snapshotRepository>
+    </distributionManagement>
+
+    <properties>
+        <apache.felix.scr.ds.annotations.version>1.2.4</apache.felix.scr.ds.annotations.version>
+        <carbon.identity.oauth.package.import.version.range>[6.0.0, 7.0.0)</carbon.identity.oauth.package.import.version.range>
+        <carbon.identity-oauth2-grant-jwt.version>1.0.31</carbon.identity-oauth2-grant-jwt.version>
+        <carbon.identity.oauth.version>6.4.80</carbon.identity.oauth.version>
+        <carbon.identity.framework.version>5.18.103</carbon.identity.framework.version>
+        <commons-lang.wso2.osgi.version.range>[2.6.0,3.0.0)</commons-lang.wso2.osgi.version.range>
+        <commons-logging.version>4.4.3</commons-logging.version>
+        <commons-logging.osgi.version.range>[1.2,2.0)</commons-logging.osgi.version.range>
+        <equinox.osgi.services.version>3.5.100.v20160504-1419</equinox.osgi.services.version>
+        <maven.surefire.plugin.version>2.21.0</maven.surefire.plugin.version>
+        <oltu.package.import.version.range>[1.0.0, 2.0.0)</oltu.package.import.version.range>
+        <oltu.version>1.0.0.wso2v3</oltu.version>
+        <imp.pkg.version.javax.servlet>[2.6.0, 3.0.0)</imp.pkg.version.javax.servlet>
+        <carbon.identity.package.import.version.range>[5.0.0, 6.0.0)</carbon.identity.package.import.version.range>
+        <commons-collections.version>3.2.2.wso2v1</commons-collections.version>
+        <commons-collections.wso2.osgi.version.range>[3.2.0,4.0.0)</commons-collections.wso2.osgi.version.range>
+        <org.wso2.securevault.version>1.0.0-wso2v2</org.wso2.securevault.version>
+        <encoder.wso2.version>1.2.0.wso2v1</encoder.wso2.version>
+        <carbon.kernel.package.import.version.range>[4.4.0, 5.0.0)</carbon.kernel.package.import.version.range>
+        <carbon.kernel.version>4.4.3</carbon.kernel.version>
+        <commons-lang.wso2.version>2.6.0.wso2v1</commons-lang.wso2.version>
+        <nimbusds.osgi.version.range>[7.3.0,8.0.0)</nimbusds.osgi.version.range>
+        <net.minidev.json.imp.pkg.version.range>[2.3.0, 3.0.0)</net.minidev.json.imp.pkg.version.range>
+        <wso2.json>3.0.0.wso2v1</wso2.json>
+        <nimbusds.version>7.3.0.wso2v1</nimbusds.version>
+        <axiom.version>1.2.11-wso2v6</axiom.version>
+        <javax.servlet-api.version>3.0-alpha-1</javax.servlet-api.version>
+        <commons-codec.version>1.4.0.wso2v1</commons-codec.version>
+        <powermock.version>1.7.0</powermock.version>
+        <jacoco.version>0.7.9</jacoco.version>
+    </properties>
+</project>
```

Comment:
add new line

Reply:
Fixed in 8693d5454cb05015e9d9614c56a1d09bdf8a8997